### ISSUE TITLE
[TEST PROF ERROR] appears when running FactoryProf, but results are correct

### ISSUE THREAD

**Issue by rtrv** [palkan/test-prof#40]
**Ruby 2.3.4**

**Rails 4.2.0, RSpec 3.1.0**

**TestProf 0.4.0**

After `FPROF=1 rspec` run I've got the following output:

```
[TEST PROF INFO] FactoryProf enabled (simple mode)
[TEST PROF ERROR]
```

However, when tests are finished, I have proper output about factories usage.

It also appears with `FPROF=flamegraph rspec`

**Comment by IDolgirev** [comment]
It caused by TestProf attempt for require `Fabrication` gem and then patch it, but when it is missing in your `Gemfile` you'll get blank log error message.

@Shkrt, will you make a PR to fix that or I can do it?

**Comment by Shkrt** [comment]
@IDolgirev thanks for hint, I'm already working on fixing it

**Comment by palkan** [comment]
Fixed and released as 0.4.1. Thanks to @Shkrt!
